sw/host/opentitanlib/src/transport/hyperdebug/mod.rs

```diff
@@ -99,88 +103,105 @@ impl Hyperdebug {
                         config_desc.number(),
                         interface.number()
                     ));
-                // Check the ASCII name of this USB interface.
-                match interface_name.as_str() {
-                    "HyperDebug Shell" => {
+                // Check the class/subclass/protocol of this USB interface.
+                if interface_desc.class_code() == 255
```

Comment:
Should these constants be given names somewhere? It seems like it might be useful to have a single struct that contains all three values so you can do a single ==

Reply:
I have given the constants names within this Hyperdebug struct.  I am not entirely convinced that doing a tuple comparison will make it more readable.